### PR TITLE
LPS-79495 SF

### PR DESCRIPTION
**Pull request by gamerson**

### CODE REVIEW

File: modules/sdk/project-templates/project-templates-activator/source-formatter-suppressions.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-activator/source-formatter-suppressions.xml:1]: Lets remove this, first we need to send a pull to SF to enable support for `Liferay-Versions` in the bnd.bnd

File: modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-api/build.gradle [hunk 2]
```diff
@@ -1,5 +1,5 @@
 dependencies {
-	compileOnly group: "biz.aQute.bnd", name: "biz.aQute.bndlib", version: "3.1.0"
+	compileOnly group: "biz.aQute.bnd", name: "biz.aQute.bndlib", version: "3.5.0"
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-api/build.gradle:2]: Lets move this to a separate commit and PR, since this is not really related, but we do need to adjust this.  So just send a new PR that updates this.

File: modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/ProjectTemplates.java [hunk 3]
```diff
@@ -54,6 +56,68 @@
 	public static final String TEMPLATE_BUNDLE_PREFIX =
 		"com.liferay.project.templates.";
 
+	public static String getManifestProperty(String property, File file)
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/ProjectTemplates.java:59]: Do add these as public statics, but just add these to Archetyper

File: modules/sdk/project-templates/project-templates/src/main/resources/com/liferay/project/templates/packageinfo [hunk 4]
```diff
@@ -1 +1 @@
-version 4.0.0
\ No newline at end of file
+version 4.1.0
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates/src/main/resources/com/liferay/project/templates/packageinfo:1]: Once you remove the other code, you wont need to bump this.

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/christopherbryanboyd/liferay-portal/pull/10. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by christopherbryanboyd** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/sdk/project-templates/project-templates-activator/source-formatter-suppressions.xml (hunk 1 above)
Comment (christopherbryanboyd) [reply modules/sdk/project-templates/project-templates-activator/source-formatter-suppressions.xml:1]: Hi @gamerson , this has been submitted to @petershin here https://github.com/petershin/liferay-portal/pull/96

File: modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-api/build.gradle (hunk 2 above)
Comment (christopherbryanboyd) [reply modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-api/build.gradle:2]: Hi @gamerson , this has been submitted to you here https://github.com/gamerson/liferay-portal/pull/113
